opentelemetry/chart/values.yaml

```diff
@@ -87,6 +59,27 @@ openTelemetry:
     additionalLabels: {}
     # plugin: kube-monitoring
     # prometheus: infra
+
+    ## Default rules for monitoring the opentelemetry components
+    rules:
+      ## Enables PrometheusRule resources to be created
+      create: true
+
+      # PrometheusRules to disable
+      disabled: []
```

Comment:
just beware, that this might break existing deployments, if they have specified a dict.